### PR TITLE
image-rs: add encrypted nydus image tests

### PR DESCRIPTION
**Pull request by ChengyuZhu6**
Add encrypted nydus image tests.

### CODE REVIEW

File: image-rs/tests/image_decryption.rs [hunk 1]
```diff
@@ -43,6 +43,8 @@ const OCICRYPT_CONFIG: &str = "test_data/ocicrypt_keyprovider_ttrpc.conf";
 #[rstest::rstest]
 #[case("ghcr.io/confidential-containers/test-container:unencrypted")]
 #[case("ghcr.io/confidential-containers/test-container:encrypted")]
+#[cfg_attr(not(feature = "nydus"), ignore)]
+#[case("ghcr.io/chengyuzhu6/busybox:encrypted-nydus")]
```
Comment (arronwy) [inline image-rs/tests/image_decryption.rs:47]: Hi @fitzthum can you help to copy this image to CoCo repo?
```
skopeo copy docker://ghcr.io/chengyuzhu6/busybox:encrypted-nydus docker://ghcr.io/confidential-containers/test-container-image-rs:busybox-encrypted-nydus
```

Comment (fitzthum) [reply image-rs/tests/image_decryption.rs:47]: Hm skopeo complains when I run that command 

```
FATA[0000] initializing image from source docker://ghcr.io/chengyuzhu6/busybox:encrypted-nydus: unsupported docker v2s2 media type: "application/vnd.oci.image.layer.nydus.blob.v1" 
```

I can take a deeper look at it tomorrow.

Comment (ChengyuZhu6) [reply image-rs/tests/image_decryption.rs]: Fixed.

Comment (ChengyuZhu6) [reply image-rs/tests/image_decryption.rs]: @fitzthum Would you kindly change the visibility of "ghcr.io/confidential-containers/busybox" to public?

Comment (fitzthum) [reply image-rs/tests/image_decryption.rs]: Whoops. Should be public now.

**Review by Xynnn007 — APPROVED** [review]
lgtm

**Review by arronwy — APPROVED** [review]
Thanks @ChengyuZhu6 LGTM!

**Pull request merged**